src/assets/fabricator/styles/components/layout.css

```diff
@@ -0,0 +1,67 @@
+.f-Layout {
+  height: 100%;
+}
+
+.f-Layout.f-is-active {
```

Comment:
It seems like we may need to namespace _everything_ at this point in the CSS because this CSS _must not ever_ conflict with project-specific CSS. So utility classes without namespace could be dangerous, right?